spec/src/consensus.rs

```diff
@@ -556,13 +665,32 @@ fn u256_low_u64(u: U256) -> u64 {
 #[cfg(test)]
 pub mod test {
     use super::*;
-    use ckb_types::core::{BlockBuilder, TransactionBuilder};
+    use ckb_resource::CODE_HASH_SECP256K1_RIPEMD160_SHA256_SIGHASH_ALL;
+    use ckb_types::{
+        core::{BlockBuilder, ScriptHashType, TransactionBuilder},
+        h160,
+        packed::Script,
+        H160,
+    };
+
+    // Satoshi's pubkey hash in Bitcoin genesis.
+    const SATOSHI_H160: H160 = h160!("0x62e907b15cbf27d5425399ebf6f0fb50ebb88f18");
 
     #[test]
     fn test_init_epoch_reward() {
         let cellbase = TransactionBuilder::default().witness(vec![].pack()).build();
         let genesis = BlockBuilder::default().transaction(cellbase).build();
-        let consensus = Consensus::new(genesis, capacity_bytes!(100));
+        let consensus = ConsensusBuilder::new(genesis, capacity_bytes!(100)).build();
         assert_eq!(capacity_bytes!(100), consensus.epoch_reward);
     }
+
+    #[test]
+    fn test_satoshi_lock_hash() {
+        let lock = Script::new_builder()
+            .code_hash(CODE_HASH_SECP256K1_RIPEMD160_SHA256_SIGHASH_ALL.pack())
+            .hash_type(ScriptHashType::Data.pack())
+            .args(vec![SATOSHI_H160.0.pack()].pack())
+            .build();
```

Comment:
This system script have type script (type id), should use `ScriptHashType::Type` instead.